src/io/pti.jl

```diff
@@ -1005,8 +1003,24 @@ function export_pti(io::IO, data::Dict{String,Any})
     end
 
     # Warnings for elements incompatibles with pti
-    Memento.warn(_LOGGER, string("Skipping storage data because is not suported in the PSSE 33 .raw file"))
-    Memento.warn(_LOGGER, string("Skipping switches data because is not suported in the PSSE 33 .raw file"))
+    incompatible_items = ["storage", "switch"]
+
+    for item in incompatible_items 
+        if haskey(data, item)
+            components = length(data[item])
+            Memento.warn(_LOGGER, string("Skipping export of the $(components) $(item) items because it is not suported in the PSSE 33 .raw file")) 
+        end
+    end
+    
+    # Warnings for not yet exported items
+    not_exported_items = ["dcline"]
+
+    for item in incompatible_items 
+        if haskey(data, item)
+            components = length(data[item])
+            Memento.warn(_LOGGER, string("Skipping $(components) $(item) data because is not yet implemented")) 
+        end
+    end
```

Comment:
PowerModels always will have dicts for all component types, but they can be empty.  So I think this should be `haskey(data, item) && length(data[item]) > 0`

Reply:
Yes. it's a good idea.
In the [PSSE Export: Code Review 2](https://github.com/lanl-ansi/PowerModels.jl/pull/753/commits/043ec834f90e996db76f12764efec643a2c256ed#diff-8eaa62afd9f02012047d607fc11deb5fc6d076a155ecf49c2ba89625666653afR1019) I put the warnings for others components that are not exported.